packages/components/forma-36-react-timepicker/tsconfig.json

```diff
@@ -20,6 +20,7 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
+    "allowJs": true,
```

Comment:
As the other comment, is there a reason to allow JS now?

Reply:
The same thing that happened to date picker happened here too, let me get the error and post it here